drivers/clock_control/Kconfig

```diff
@@ -9,6 +9,13 @@
 #
 # Clock controller drivers
 #
+
+config CLOCK_CONTROL_MUST_USE_OUT_OF_TREE_DRIVER
+  bool
+  help
+	  This option can be enabled by an out-of-tree library if it
+	  needs to take control over the clock controller.
+
```

Comment:
Weird formatting. Check how other symbols are formatted, with tabs vs. spaces, etc.

Reply:
It is inconsistent across many, many files unfortunately. As far as I've seen, most places use tab + 2x space:
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/clock_control/Kconfig.nrf#L11
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/counter/Kconfig.qmsi#L13
https://github.com/zephyrproject-rtos/zephyr/blob/master/kernel/Kconfig#L20

Reply:
Thinking about before the `bool` and the `help`. There's two spaces there now instead of a tab.